sw/device/lib/testing/flash_ctrl_testutils.h

```diff
@@ -274,10 +274,33 @@ void flash_ctrl_testutils_set_counter(dif_flash_ctrl_state_t *flash_state,
  * This function can be used to initialize a NVM counter to zero by filling
  * its flash region with non-zero values.
  *
- * @param flash_state A flash_ctrl handle
- * @param counter The ID of the NVM counter, [0, 2]
+ * @param flash_state A flash_ctrl handle.
+ * @param counter The ID of the NVM counter, [0, 2].
  **/
 void flash_ctrl_testutils_counter_init_zero(dif_flash_ctrl_state_t *flash_state,
                                             size_t counter);
 
+/**
+ * Init the backdoor API, it should be called before
+ * `flash_ctrl_testutils_backdoor_wait_update`.
+ *
+ * @param flash_state A flash_ctrl handle.
+ */
+void flash_ctrl_testutils_backdoor_init(dif_flash_ctrl_state_t *flash_state);
+
+/**
+ * This is a backdoor API to be used with dvsim testbench.
+ * Backdoors variables seating on flash may not be updated when read by software
```

Comment:
Perhaps 'Backdoor variables present in flash'?

Reply:
Fixed